### ISSUE TITLE
SpongeForge is hard dependency on prior 1.12.2

### ISSUE THREAD

**Issue by HowardZHY** [CleanroomMC/MixinBooter#81]
https://github.com/CleanroomMC/MixinBooter/commit/1fd9eeaa04606adc3809c555627b349ebbef7016

This commit made SpongeForge REQUIRED to be installed on older Forge

https://github.com/CleanroomMC/MixinBooter/issues/69

**Comment by KorewaLidesu** [comment]
Do you have a log that present the error?
Please upload it via mslo.gs

**Comment by HowardZHY** [comment]
https://mclo.gs/2QKMpjz
<img width="884" alt="image" src="https://github.com/CleanroomMC/MixinBooter/assets/96782166/200e3e2c-598b-4da4-828c-f76dd8f1bb9c">

The only errors are
```[22:30:33] [main/FATAL] [FML/]: The mod mixinbooter (MixinBooter) requires mods [spongeforge] to be available
[22:30:33] [main/FATAL] [FML/]: Mod mixinbooter (MixinBooter) requires [spongeforge@[7.4.8,)]```

> Do you have a log that present the error? Please upload it via mslo.gs

**Comment by Rongmario** [comment]
Fixed in 558f135
